README.md

```diff
@@ -258,7 +258,7 @@ The previous section would have generated a debug version of the app. In order t
 First, if you do not already have a keystore, create it as follows (for more information, see the [Android documentation](https://developer.android.com/studio/publish/app-signing#signing-manually)):
 
 ```
-nix-shell -p androidenv.platformTools --run "keytool -genkey -v -keystore myandroidkey.jks -keyalg RSA -keysize 2048 -validity 10000 -alias myandroidalias"
+nix-shell -E "let pkgs = import <nixpkgs> {}; in pkgs.mkShell { buildInputs = [ pkgs.jdk9 ]; }" --run "keytool -genkey -v -keystore myandroidkey.jks -keyalg RSA -keysize 2048 -validity 10000 -alias myandroidalias"
```

Comment:
better to use the project's nixpkgs here to make this doc more reproducible

Reply:
This is a really complex command to ask the user to do.  Can we give them an `ob` command or make it an automatic part of deployment somehow?

Reply:
Actually this whole section is outdated now that `obelisk` generates the keystore/key for the user. The user only gets prompted for a keystore password during deploy command and everything else gets set with default values that get tracked from within config directory. Though it would still be useful to have documented how to get an environment with `keytool` in order to make tweaks from obelisk's default values like switching alias or anything else

Reply:
@hSloan @luigy Based on this discussion, should we get rid of this stuff?